doc/whats_new/v0.20.rst

```diff
@@ -29,6 +29,7 @@ random sampling procedures.
 - :class:`neural_network.MLPRegressor` (bug fix)
 - :class:`neural_network.MLPClassifier` (bug fix)
 - :class:`neural_network.BaseMultilayerPerceptron` (bug fix)
+- :class:`ensemble.gradient_boosting.BaseGradientBoosting` (bug fix)
```

Comment:
I think we should probably list the public classifier/regressor rather than Base. We should also note that it only affects feature importances